Makefile

```diff
@@ -67,6 +67,8 @@ install-user: venv-create
 install: install-user
 	# Also install development dependencies
 	. $(VENV_ACTIVATE_FILE); $(PIP_WRAPPER) install -e .[develop]
+	. $(VENV_ACTIVATE_FILE); $(PIP_WRAPPER) install git+https://github.com/elastic/pytest-rally.git
```

Comment:
I think it's a bug in pip. In any case, I think we should upload `pytest-rally` on PyPI since it's not expected to change much and the resulting experience would be nicer.

Reply:
Yeah, I looked into it a bit and it does see to be a bug in `pip`. Definitely agree on uploading the plugin to PyPI once it's stable.